<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelInputFile">
      <property name="text">
       <string>Input Text File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputFileLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="browseTextButton">
      <property name="minimumSize">
       <size>
        <width>1136</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="compressButtonlabel">
      <property name="minimumSize">
       <size>
        <width>1136</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Compress</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelCompressedFil">
      <property name="text">
       <string>Compressed File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="compressedFileLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="browseCompressedButton">
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decompressButton">
      <property name="text">
       <string>Decompress</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downloadCompressedButton">
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelDecompressedFile">
      <property name="text">
       <string>Decompressed File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="decompressedFileLineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downloadDecompressedButton">
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
